cmd/internal/collector/collector.go

```diff
@@ -189,3 +196,14 @@ func NewCCloudCollector() CCloudCollector {
 
 	return collector
 }
+
+func matchesRegex (candidate string, regex string) bool {
+	matched, err := regexp.MatchString(regex,candidate)
```

Comment:
It seems that there is a space missing here `regexp.MatchString(regex, candidate)`
You should probably do a `go fmt` to format the source code ;)